src/pages/bots/[id]/index.tsx

```diff
@@ -56,7 +56,11 @@ const ChatThread = (props: {
                 <ul className="list-disc px-8">
                   {sourcesLinks?.map((link) => (
                     <li key={link}>
-                      <a className="text-blue-600 font-bold" href={link ?? ''}>
+                      <a
+                        className="text-blue-600 font-bold"
+                        href={link ?? ''}
+                        target="_blank"
```

Comment:
Should add `rel="noopener noreferrer"` to avoid hijacking. Not sure if all major browsers handle it by default. Google to decide if we still need the attributes or not.

```suggestion
                        target="_blank"
                        rel="noopener noreferrer"
```

Reply:
It's a good lesson for me in security.